src/Controls/src/Core/FontElement.cs

```diff
@@ -6,26 +6,31 @@ static class FontElement
 	{
 		public static readonly BindableProperty FontProperty =
 			BindableProperty.Create("Font", typeof(Font), typeof(IFontElement), default(Font),
+									propertyChanging: OnFontChanging,
 									propertyChanged: OnFontPropertyChanged);
```

Comment:
I think that maybe we are supposed to use this value in the various Font properties. Not sure why we have this AND the field. Not only is it being cached here, but we don't have 2 caches - which is probably causing the issues that you are seeing.